Storage/src/Connection/Rest.php

```diff
@@ -600,4 +602,28 @@ protected function supportsBuiltinCrc32c()
     {
         return Builtin::supports(CRC32::CASTAGNOLI);
     }
+
+    /**
+     * Add the required retry function and send the request.
+     *
+     * @param string $resource resource name, eg: buckets.
+     * @param string $method method name, eg: get
+     * @param array $args
+     */
+    private function sendRequest($resource, $method, $args)
```

Comment:
It'd be better to name this something more descriptive, such as `sendWithRetry`

Reply:
Sure, Done!